src/inverseRobot.js

```diff
@@ -24,7 +24,26 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const properties = Object.values(robot);
+  const values = Object.keys(robot);
+
+  for (let i = 0; i < properties.length; i++) {
+    for (let j = 0; j < values.length; j++) {
+      if (i !== j) {
+        if (properties[i] === properties[j] || values[i] === values[j]) {
+          return null;
+        }
+      }
+    }
+  }
```

Comment:
Omg🤯 Here is 2 nested loops.

Let's find a simpler way how to do it.

Hint: In this task, you need only 1 loop. You should use `in` or `hasOwnProperty` to check if a property exists in an object